### PR TITLE
Continue migrating #135 (letter 'r')

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
This PR should be good to go

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4767,52 +4914,94 @@ repulican->republican
...
+resetted->reset
 resevoir->reservoir
+resgister->register
+resgisters->registers
 residental->residential
 resignement->resignment
 resistable->resistible
 resistence->resistance
 resistent->resistant
+resizeable->resizable
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: resizeable is valid http://www.yourdictionary.com/resizeable

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: please remove then

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Done. I changed  
`resizeable->resizable`  
to  
`resizeble->resizable`

**Pull request merged**